devops/jobs/CreateSandboxSSHAccess.groovy

```diff
@@ -70,6 +70,8 @@ class CreateSandboxSSHAccess {
         sshAgent('sandbox-ssh-keys')
       }
 
+      authenticationToken("${JENKINS_AUTH_TOKEN}")
```

Comment:
According to Stu, there is a [End-to-End Job](https://github.com/edx/jenkins-job-dsl-internal/blob/master/jobs/tools-edx-jenkins.edx.org/createEndToEndTestJobs.groovy#L49) that can be called remotely via an auth token that set in the highlighted line. If you look at line 36, a "${JENKINS_AUTH_TOKEN}" is passed as a parameter to the Ecommerce Config (linked hashmap) above. I've tried "grepping" the code for this token but no success (must be an environmental variable)

Reply:
Decided to store a "JENKINS_SANDBOX_JOB_KEY" variable (here's the [PR](https://github.com/edx/edx-internal/pull/1023) that will be read using extraVars.get(xxx)